scripts/src/main/resources/scripts/command/helm

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Check if helm is installed.
+function doIsHelmInstalled() {
+  if command -v helm &> /dev/null
+  then
+    return
+  else
+    return 255
+  fi
+}
+
+# Call helm with specified arguments.
+function doRun() {
+  doSetup silent
+  helm "${@}"
+}
+
+function doSetup() {
+
+  if doIsHelmInstalled
+  then
+    if [ "${1}" != "silent" ] && ! doIsQuiet
+    then
+      helm version
```

Comment:
same issue(s) as with terraform.
Can you move the version output to the end (any consider using `doRunCommand` that will also add automatic error handling)?